### PR TITLE
Tweaked markup item account name

### PR DESCRIPTION
**Pull request by greycampbell**

### CODE REVIEW

File: lib/netsuite/records/markup_item.rb [hunk 1]
```diff
@@ -19,12 +19,11 @@ class MarkupItem
 
       record_refs :klass,
                   :department,
-                  :expense_account,
+                  :account,
                   :location,
                   :tax_schedule
 
       field :custom_field_list, CustomFieldList
```
Comment (madelinewu) [inline lib/netsuite/records/markup_item.rb:26]: does item vendor list not exist?

Comment (madelinewu) [reply lib/netsuite/records/markup_item.rb:26]: oops think it was my bad!

**Review by madelinewu — APPROVED** [review]

File: lib/netsuite/records/markup_item.rb (hunk 1 above)
Comment (greycampbell) [reply lib/netsuite/records/markup_item.rb:26]: I didn't see it in NS [here](https://7232422-sb1.app.netsuite.com/app/common/item/item.nl?id=16&e=T)

Comment (greycampbell) [reply lib/netsuite/records/markup_item.rb:26]: Only vendor name which I don't think is really the same, its freeform text vs a vendor ref

Comment (madelinewu) [reply lib/netsuite/records/markup_item.rb:26]: ya sorry - i meant that i checked NS as well and realized i messed up earlier!

Comment (greycampbell) [reply lib/netsuite/records/markup_item.rb:26]: Oh gotcha! no worries at all

**Pull request merged**